src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.cs

```diff
@@ -390,6 +395,10 @@ internal partial class UpdateVerbParser
                         "--configfile",
                         Strings.Option_ConfigFile,
                         CommandOptionType.SingleValue);
+                    CommandOption allowInsecureConnections = SourceCmd.Option(
+                        "--allowInsecureConnections",
```

Comment:
Options in dotnet are all lower case and snake case. 

See: store-password-in-clear-text, valid-authentication-types etc.